packages/mermaid/src/mermaidAPI.ts

```diff
@@ -511,7 +511,9 @@ const render = async function (
   try {
     await diag.renderer.draw(text, id, version, diag);
   } catch (e) {
-    errorRenderer.draw(text, id, version);
+    if (!config.suppressErrorRendering) {
+      errorRenderer.draw(text, id, version);
+    }
```

Comment:
Error diagram is initialized on line 479 too. 
Please verify that.

Reply:
Thanks. I fixed this at b4618f9